emperor/support_files/js/emperor.js

```diff
@@ -873,7 +874,7 @@ function displayTaxaLabels(taxaLevel){
 
         $('#taxalabels').css('display','block');
         $("#taxaLevel" ).html(taxonomy[taxaLevel]);
-        for(var key in g_taxaPositions){
+        _.each(_.range(g_taxaPositions.length),function(key){
```

Comment:
Another way to do this is by:
_.each(g_taxaPositions, function(your_object, key){... })

_each will return 1: the object, 2: the enumerate, 3: the full object.